lib/ProductOpener/Display.pm

```diff
@@ -10557,6 +10558,7 @@ sub display_product_history ($request_ref, $code, $product_ref) {
 			number => $change_rev,
 			date => display_date_tag($change_ref->{t}),
 			userid => $userid,
+            uuid => $uuid,
```

Comment:
Now that you have added that variable, you can display it in the template: template/web/pages/product/includes/edit_history.tt.html

Reply:
updated it, thank you @stephanegigandet